docs/process/releasing-updates.md

```diff
@@ -6,25 +6,60 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 - `production` is the channel from which the general public downloads and receives updates. It should be stable and polished.
 
-- `beta` is released more often than `production`. It may be buggy and unpolished.
+- `beta` is released more often than `production`. It may be buggy and unpolished. For example: we should be able to release at any point from the master branch.
 
 - `test` is unlike the other two. It does not receive updates. Each test release is locked in time. It's used entirely for providing test releases.
 
 ## The Process
 
-From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and run:
+For releasing to `test`, skip to the `Releasing` header (no draft release needed).
```

Comment:
This is accurate in that `test` isn't supported in the `draft-release` command, but you still need to bump the version and add a changelog entry. If we want to add in support for that (and I think we should, because it's super unclear) let's track it as a `tech-debt` task.